data-access/bestuursperiode.ts

```diff
@@ -0,0 +1,61 @@
+import { sparqlEscapeString } from 'mu';
+import { querySudo } from '@lblod/mu-auth-sudo';
+import {
+  findFirstSparqlResult,
+  getBooleanSparqlResult,
+} from '../util/sparql-result';
+import { HttpError } from '../util/http-error';
+import { STATUS_CODE } from '../util/constants';
+
+export const bestuursperiode = {
+  exists,
+  getIdForUri,
+};
+
+async function exists(bestuursperiodeId: string): Promise<boolean> {
+  const askIfExists = `
+    PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+    PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+
+      ASK {
+        GRAPH ?bestuursperiodeGraph {
+          ?bestuursperiode a ext:Bestuursperiode;
+            mu:uuid ${sparqlEscapeString(bestuursperiodeId)}.
+        }
+
+        FILTER ( ?bestuursperiodeGraph != <http://mu.semte.ch/vocabularies/ext/FormHistory> )
```

Comment:
wrong filter, also you don't really need this, bestuursperiodes are always written in public.